marklogic-data-hub/src/test/resources/entity-migration-test/entities/Customer.entity.json

```diff
@@ -0,0 +1,97 @@
+{
+  "info": {
+    "title": "Customer",
+    "version": "0.0.1",
+    "baseUri": "http://example.org/"
+  },
+  "definitions": {
+    "Customer": {
+      "elementRangeIndex": [ "nicknames", "status" ],
+      "rangeIndex": [ "name", "birthDate" ],
```

Comment:
Need to add pathRangeIndex. I recommend adding some "bad" entity files to the project too - ones that are missing info.title, or where the value of info/title is not found in "definitions".